journalapp/scripts/get_crisewing_api.py

```diff
@@ -0,0 +1,81 @@
+"""Script to retrieve existing journal entries in Cris Ewing's database."""
+
+import os
+import sys
+import requests
+from journalapp.scripts import initializedb
+from datetime import datetime
+from journalapp.models import Entry, DBSession
+import transaction
+
+DATETIME_FORMAT = '%Y-%m-%dT%H:%M:%S.%f'
+
+
+def get_api_key():
+    """Return API key from system global environmental variables."""
+    key = os.environ.get('CRISEWING_API_KEY')
+    if not key:
+        print('crisewing API key not found in os.environ.')
+        sys.exit()
```

Comment:
goood!  However, when calling `sys.exit` it is traditional (and considered good form) to include an integer return value.  In general a 0 or empty return value is interpreted by most shell programs as success, so if you want to signal that something went wrong, send a non-zero integer:

```python
sys.exit(1)
```